src/comments.js

```diff
@@ -381,6 +383,20 @@ function handleConditionalExpressionComments(enclosingNode, followingNode, comme
   return false;
 }
 
+function handleAssignmentPatternComments(enclosingNode, comment) {
```

Comment:
But, funny thing, switching to `comment.printed = true` doesn't break any tests 😮! Then, I just need to figure out how to print the comment for:

```js
const {a /* comment */ = 1 } = b
```